api/src/main/java/com/datastrato/gravitino/file/FilesetChange.java

```diff
@@ -69,6 +69,15 @@ static FilesetChange removeProperty(String property) {
     return new RemoveProperty(property);
   }
 
+  /**
+   * Creates a new fileset change to remove comment from the fileset.
+   *
+   * @return The fileset change.
+   */
+  static FilesetChange removeComment() {
+    return new RemoveComment();
```

Comment:
Since there are no parameters in the constructor, we can return a static result to avoid creating a new instance every time.

Reply:
fix